src/include/duckdb/common/serializer/deserializer.hpp

```diff
@@ -94,6 +116,25 @@ class Deserializer {
 		OnPropertyEnd();
 	}
 
+	// Read a property and discard the value
+	template <typename T>
+	inline void ReadDeletedProperty(const field_id_t field_id, const char *tag) {
+		(void)ReadProperty<T>(field_id, tag);
```

Comment:
Shouldn't a deleted property always be optional? Otherwise we need to keep writing it to the file, no?